config/proto/probe.proto

```diff
@@ -61,8 +61,17 @@ message HermesProbeDef {
   cloudprober.metrics.Dist probe_latency_distribution = 7;  // Measures the latency of Hermes' probes, allowing you to diagnose a network issue or issue with Hermes.
   cloudprober.metrics.Dist api_call_latency_distribution = 8;  // Measures the latency of the API calls for the relevant storage system targeted.
 
-  // Valid values: "ns", "us" (or "µs"), "ms", "s", "m", "h".
-  string latency_unit = 9;  // default is "us"
+  enum latencyUnit {
```

Comment:
Nit: s/latencyUnit/LatencyUnit/.
https://developers.google.com/protocol-buffers/docs/style#enums

Reply:
Question about this comment: is s/latencyUnit/LatencyUnit/. shorthand for:
Change latencyUnit to LatencyUnit

If so, I'm happy to change this, thanks for pointing it out.